Margin-pad-fixes

efuenzalida52 opened the pull request:
Added styling for margin in alignstandard div

efuenzalida52 committed:
LMC Catalog Search Box Block has no bottom margin #281

Added styling for margin in alignstandard div

efuenzalida52 committed:
LMC Catalog Search Box Block has no bottom margin #281

Shortcode FormAssembly forms have no bottom margin #282

Added margin styling to respective scss divs.

efuenzalida52 committed:
LMC Catalog Search Box Block has no bottom margin #281

added auto

taija commented on src/scss/main.scss:
```diff
@@ -58,6 +58,7 @@ $header-desktop-breakpoint: lg;
 		&.alignstandard>*,
 		&.aligncenter {
 			max-width: var(--wp--style--global--content-size);
+			margin: var(--sitka-container-spacing) auto;
```
@efuenzalida52 I'm a bit concerned about potential impacts of this, since it's adding it to all elements with any alignment, not just the elements you were working on. @TaylorYeeBC  - make sure we test this.

efuenzalida52 committed:
Created New Stylesheet for lmc styling

This is to prevent wonkyness in editing general align styles that apply to a wide range.

taija commented on src/scss/components/_formassembly.scss:
```diff
@@ -5,7 +5,7 @@
 	.wFormContainer {
 		max-width: 1440px;
-		margin: 0 auto;
+		margin: var(--sitka-container-spacing) auto;
```
This will add margin on both the top *and* bottom of the form; @TaylorYeeBC  double check this on it's own, and make sure that it doesn't end up with more spacing than it should

TaylorYeeBC replied:
Extra spacing was added to the top, you are correct.
![image](https://github.com/user-attachments/assets/6a80ea12-0031-4246-9bad-141eb3e34a06)

efuenzalida52 committed:
Correcting bottom margin  in mobile views issue

taija reviewed (approved):
Look good to me (code review, not test pass)

efuenzalida52 committed:
adjusting forms assembly margin fix

efuenzalida52 committed:
reverted changes so mobile is working again & fixed alignment issue

TaylorYeeBC reviewed (approved):
Margins in web/mobile views are within acceptable boundaries and are consistent with rest of theme. Marking as approved.
